src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useRef } from 'react'
+
+import { Modal, ModalRef } from './Modal'
+import { ModalHeading } from './ModalHeading/ModalHeading'
+import { ModalFooter } from './ModalFooter/ModalFooter'
+import { ModalToggleButton } from './ModalToggleButton'
+
+import { ButtonGroup } from '../ButtonGroup/ButtonGroup'
+
+export default {
+  title: 'Components/Modal',
+  component: Modal,
+  parameters: {
+    docs: {
+      description: {
+        component: `
+### USWDS 2.0 Modal component
+
+Source: http://designsystem.digital.gov/components/modal
+
+To use this component, you will need to create a ref and pass it into the rendered Modal component. This ref will expose several properties (also described by the exported ModalRef type):
+
+- modalId: string
+  - the value of the id attribute given to the modal
+- modalIsOpen: boolean
+  - true if the modal is currently open, otherwise false
+- toggleModal: (e?: React.MouseEvent, open?: boolean) => boolean
+  - use this function to open or close the modal.
+  - if attached to an event handler, pass the event in
+  - if the second argument is provided, it will explicitly open the modal if true, or explicitly close it if false
+  - returns true if the toggle operation was successful, false if the event was prevented.
+
+Follow the [USWDS](https://designsystem.digital.gov/components/modal/) guidance for using modals:
+
+- Pass a unique ID into each modal component.
+- Any component that opens the modal should be a button or anchor element, have the [data-open-modal] attribute, and [aria-controls] attribute with the value of the modal ID.
+- Any component that closes the modal should be a button element, and have the [data-close-modal] attribute, and [aria-controls] attribute with the value of the modal ID.
+- Use the forceAction prop on the modal component if the user should be forced to take an action before closing the modal.
+
+You can also use the provided ModalToggleButton and/or ModalOpenLink components, which will adhere to the above guidelines for convenience.
+`,
+      },
+    },
+  },
+}
+
+export const defaultModal = (): React.ReactElement => {
```

Comment:
I was looking at the Happo diff and I'm curious if you think it would be valuable to have a story for a modal that is in open state by default. That way Happo is able to detect changes to the modal content rather than just the toggle button, which is all we can currently see.

Reply:
oh yes good call! I can add stories just for the ModalWindow component. I'll do that!